### PR TITLE
documented meetup architecture

### PR DESCRIPTION
**Pull request by exploreshaifali**
I believe one separate file containing detailed architecture of meetup is also needed but better if we add it after implementing permissions. What say?

### CODE REVIEW

**Comment by ana-balica** [comment]
Permissions will take some time (pretty sure about that). So if you think we need first permission, let's just postpone the docs. We still have plenty of docstrings explaining what is happening.

File: docs/develop/architecture.rst [hunk 1]
```diff
@@ -16,6 +16,9 @@ As any other Django project Portal organizes its functionality in several apps:
   requests to a community, removing and inviting users to become members of a
   community.
 * **users** - showing and editing user personal profile.
+* **meetup** - handles meetup locations and meetup functionality, like adding, showing, editing,
```
Comment (ana-balica) [inline docs/develop/architecture.rst]: Make it shorter - "handles meetup locations and meetup functionality" - that will probably we enough as an overview.

Comment (exploreshaifali) [inline docs/develop/architecture.rst]: Done.

**Comment by ana-balica** [comment]
Merged :)

**Pull request closed without merging**